{"uid":"e73cf070e8a2a742","name":"TC 03.02.01 to 03_02_10 Verify Failed Login, no error validation","fullName":"tests.test_sign_in.TestFailedLogin#test_03_02_01_to_03_02_10_login","historyId":"610d546b6c84a684ec5e86056fce9b5f","time":{"start":1737857787751,"stop":1737857793291,"duration":5540},"description":"Check Success and Failed Login, no error validation","descriptionHtml":"<p>Check Success and Failed Login, no error validation</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5642925f27ca <unknown>\n#1 0x5642920ea2f0 <unknown>\n#2 0x56429213a035 <unknown>\n#3 0x56429213a251 <unknown>\n#4 0x564292180054 <unknown>\n#5 0x56429215e9dd <unknown>\n#6 0x56429217d3b3 <unknown>\n#7 0x56429215e753 <unknown>\n#8 0x56429212bbaa <unknown>\n#9 0x56429212cdfe <unknown>\n#10 0x5642925bd38b <unknown>\n#11 0x5642925c1307 <unknown>\n#12 0x5642925a9e7c <unknown>\n#13 0x5642925c1ec7 <unknown>\n#14 0x56429258e24f <unknown>\n#15 0x5642925e12f8 <unknown>\n#16 0x5642925e14c0 <unknown>\n#17 0x5642925f1646 <unknown>\n#18 0x7f5e15e9ca94 <unknown>\n#19 0x7f5e15f29c3c <unknown>","statusTrace":"self = <test_sign_in.TestFailedLogin object at 0x7ff9a15df910>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"bcc45957cbc345abb578d76f92af6c66\")>\nemail = ' teston120@mailinator.com ', password = 'Test#12p@ssword'\n\n    @allure.title('TC 03.02.01 to 03_02_10 Verify Failed Login, no error validation')\n    @pytest.mark.parametrize('email, password', LOGIN)\n    def test_03_02_01_to_03_02_10_login(self, driver, email, password):\n        \"\"\"Check Success and Failed Login, no error validation\"\"\"\n        sign_in_page = SignInPage(driver, SIGN_IN_URL)\n        sign_in_page.open()\n>       sign_in_page.fill_in_email_field(email)\n\ntests/test_sign_in.py:256: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/sign_in_page.py:54: in fill_in_email_field\n    self.fill_in_field(self.check_customer_email_field_is_clickable(), email)\npages/sign_in_page.py:41: in check_customer_email_field_is_clickable\n    return self.element_is_clickable(self.locators.CUSTOMER_EMAIL)\npages/base_page.py:80: in element_is_clickable\n    return wait(self.driver, timeout).until(EC.element_to_be_clickable(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"bcc45957cbc345abb578d76f92af6c66\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7ff9a2bbd080>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5642925f27ca <unknown>\nE       #1 0x5642920ea2f0 <unknown>\nE       #2 0x56429213a035 <unknown>\nE       #3 0x56429213a251 <unknown>\nE       #4 0x564292180054 <unknown>\nE       #5 0x56429215e9dd <unknown>\nE       #6 0x56429217d3b3 <unknown>\nE       #7 0x56429215e753 <unknown>\nE       #8 0x56429212bbaa <unknown>\nE       #9 0x56429212cdfe <unknown>\nE       #10 0x5642925bd38b <unknown>\nE       #11 0x5642925c1307 <unknown>\nE       #12 0x5642925a9e7c <unknown>\nE       #13 0x5642925c1ec7 <unknown>\nE       #14 0x56429258e24f <unknown>\nE       #15 0x5642925e12f8 <unknown>\nE       #16 0x5642925e14c0 <unknown>\nE       #17 0x5642925f1646 <unknown>\nE       #18 0x7f5e15e9ca94 <unknown>\nE       #19 0x7f5e15f29c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1737855487853,"stop":1737855487915,"duration":62},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_allure_result_folder","time":{"start":1737855487916,"stop":1737855487916,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1737857786569,"stop":1737857787750,"duration":1181},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Check Success and Failed Login, no error validation","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5642925f27ca <unknown>\n#1 0x5642920ea2f0 <unknown>\n#2 0x56429213a035 <unknown>\n#3 0x56429213a251 <unknown>\n#4 0x564292180054 <unknown>\n#5 0x56429215e9dd <unknown>\n#6 0x56429217d3b3 <unknown>\n#7 0x56429215e753 <unknown>\n#8 0x56429212bbaa <unknown>\n#9 0x56429212cdfe <unknown>\n#10 0x5642925bd38b <unknown>\n#11 0x5642925c1307 <unknown>\n#12 0x5642925a9e7c <unknown>\n#13 0x5642925c1ec7 <unknown>\n#14 0x56429258e24f <unknown>\n#15 0x5642925e12f8 <unknown>\n#16 0x5642925e14c0 <unknown>\n#17 0x5642925f1646 <unknown>\n#18 0x7f5e15e9ca94 <unknown>\n#19 0x7f5e15f29c3c <unknown>","statusTrace":"self = <test_sign_in.TestFailedLogin object at 0x7ff9a15df910>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"bcc45957cbc345abb578d76f92af6c66\")>\nemail = ' teston120@mailinator.com ', password = 'Test#12p@ssword'\n\n    @allure.title('TC 03.02.01 to 03_02_10 Verify Failed Login, no error validation')\n    @pytest.mark.parametrize('email, password', LOGIN)\n    def test_03_02_01_to_03_02_10_login(self, driver, email, password):\n        \"\"\"Check Success and Failed Login, no error validation\"\"\"\n        sign_in_page = SignInPage(driver, SIGN_IN_URL)\n        sign_in_page.open()\n>       sign_in_page.fill_in_email_field(email)\n\ntests/test_sign_in.py:256: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/sign_in_page.py:54: in fill_in_email_field\n    self.fill_in_field(self.check_customer_email_field_is_clickable(), email)\npages/sign_in_page.py:41: in check_customer_email_field_is_clickable\n    return self.element_is_clickable(self.locators.CUSTOMER_EMAIL)\npages/base_page.py:80: in element_is_clickable\n    return wait(self.driver, timeout).until(EC.element_to_be_clickable(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"bcc45957cbc345abb578d76f92af6c66\")>\nmethod = <function element_to_be_clickable.<locals>._predicate at 0x7ff9a2bbd080>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x5642925f27ca <unknown>\nE       #1 0x5642920ea2f0 <unknown>\nE       #2 0x56429213a035 <unknown>\nE       #3 0x56429213a251 <unknown>\nE       #4 0x564292180054 <unknown>\nE       #5 0x56429215e9dd <unknown>\nE       #6 0x56429217d3b3 <unknown>\nE       #7 0x56429215e753 <unknown>\nE       #8 0x56429212bbaa <unknown>\nE       #9 0x56429212cdfe <unknown>\nE       #10 0x5642925bd38b <unknown>\nE       #11 0x5642925c1307 <unknown>\nE       #12 0x5642925a9e7c <unknown>\nE       #13 0x5642925c1ec7 <unknown>\nE       #14 0x56429258e24f <unknown>\nE       #15 0x5642925e12f8 <unknown>\nE       #16 0x5642925e14c0 <unknown>\nE       #17 0x5642925f1646 <unknown>\nE       #18 0x7f5e15e9ca94 <unknown>\nE       #19 0x7f5e15f29c3c <unknown>\n\n/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[{"name":"Open a browser","time":{"start":1737857787751,"stop":1737857788173,"duration":422},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Fill in Email field","time":{"start":1737857788174,"stop":1737857793290,"duration":5116},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5642925f27ca <unknown>\n#1 0x5642920ea2f0 <unknown>\n#2 0x56429213a035 <unknown>\n#3 0x56429213a251 <unknown>\n#4 0x564292180054 <unknown>\n#5 0x56429215e9dd <unknown>\n#6 0x56429217d3b3 <unknown>\n#7 0x56429215e753 <unknown>\n#8 0x56429212bbaa <unknown>\n#9 0x56429212cdfe <unknown>\n#10 0x5642925bd38b <unknown>\n#11 0x5642925c1307 <unknown>\n#12 0x5642925a9e7c <unknown>\n#13 0x5642925c1ec7 <unknown>\n#14 0x56429258e24f <unknown>\n#15 0x5642925e12f8 <unknown>\n#16 0x5642925e14c0 <unknown>\n#17 0x5642925f1646 <unknown>\n#18 0x7f5e15e9ca94 <unknown>\n#19 0x7f5e15f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/sign_in_page.py\", line 54, in fill_in_email_field\n    self.fill_in_field(self.check_customer_email_field_is_clickable(), email)\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/sign_in_page.py\", line 41, in check_customer_email_field_is_clickable\n    return self.element_is_clickable(self.locators.CUSTOMER_EMAIL)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 80, in element_is_clickable\n    return wait(self.driver, timeout).until(EC.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Check Email field is clickable","time":{"start":1737857788174,"stop":1737857793290,"duration":5116},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5642925f27ca <unknown>\n#1 0x5642920ea2f0 <unknown>\n#2 0x56429213a035 <unknown>\n#3 0x56429213a251 <unknown>\n#4 0x564292180054 <unknown>\n#5 0x56429215e9dd <unknown>\n#6 0x56429217d3b3 <unknown>\n#7 0x56429215e753 <unknown>\n#8 0x56429212bbaa <unknown>\n#9 0x56429212cdfe <unknown>\n#10 0x5642925bd38b <unknown>\n#11 0x5642925c1307 <unknown>\n#12 0x5642925a9e7c <unknown>\n#13 0x5642925c1ec7 <unknown>\n#14 0x56429258e24f <unknown>\n#15 0x5642925e12f8 <unknown>\n#16 0x5642925e14c0 <unknown>\n#17 0x5642925f1646 <unknown>\n#18 0x7f5e15e9ca94 <unknown>\n#19 0x7f5e15f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/sign_in_page.py\", line 41, in check_customer_email_field_is_clickable\n    return self.element_is_clickable(self.locators.CUSTOMER_EMAIL)\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 80, in element_is_clickable\n    return wait(self.driver, timeout).until(EC.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Find clickable elements","time":{"start":1737857788174,"stop":1737857793290,"duration":5116},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x5642925f27ca <unknown>\n#1 0x5642920ea2f0 <unknown>\n#2 0x56429213a035 <unknown>\n#3 0x56429213a251 <unknown>\n#4 0x564292180054 <unknown>\n#5 0x56429215e9dd <unknown>\n#6 0x56429217d3b3 <unknown>\n#7 0x56429215e753 <unknown>\n#8 0x56429212bbaa <unknown>\n#9 0x56429212cdfe <unknown>\n#10 0x5642925bd38b <unknown>\n#11 0x5642925c1307 <unknown>\n#12 0x5642925a9e7c <unknown>\n#13 0x5642925c1ec7 <unknown>\n#14 0x56429258e24f <unknown>\n#15 0x5642925e12f8 <unknown>\n#16 0x5642925e14c0 <unknown>\n#17 0x5642925f1646 <unknown>\n#18 0x7f5e15e9ca94 <unknown>\n#19 0x7f5e15f29c3c <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 80, in element_is_clickable\n    return wait(self.driver, timeout).until(EC.element_to_be_clickable(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.11/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[{"name":"locator","value":"('css selector', '#email')"},{"name":"timeout","value":"5"}],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[{"name":"email","value":"' teston120@mailinator.com '"}],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"70403239f0a8c446","name":"Screenshot","source":"70403239f0a8c446.png","type":"image/png","size":25650},{"uid":"f5461020e65204aa","name":"HTML source","source":"f5461020e65204aa.html","type":"text/html","size":258925}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":4,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1737857793428,"stop":1737857793492,"duration":64},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_allure_result_folder::0","time":{"start":1737857973906,"stop":1737857973906,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"feature","value":"Failed Login Functionality"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_sign_in"},{"name":"subSuite","value":"TestFailedLogin"},{"name":"host","value":"fv-az1314-617"},{"name":"thread","value":"2774-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_sign_in"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"email","value":"' teston120@mailinator.com '"},{"name":"password","value":"'Test#12p@ssword'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"e73cf070e8a2a742.json","parameterValues":["' teston120@mailinator.com '","'Test#12p@ssword'"]}